### PR TITLE
Add remove & updateAnnotation methods for Android.

### PR DESCRIPTION
**Pull request by christopherdro**

### CODE REVIEW

**Comment by bsudekum** [comment]
This looks great! Can you also update example.js  and api.md?

File: index.android.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ var MapMixins = {
   addAnnotations(mapRef, annotations) {
     NativeModules.MapboxGLManager.addAnnotations(React.findNodeHandle(this.refs[mapRef]), annotations);
   },
+  updateAnnotation(mapRef, annotation) {
+    NativeModules.MapboxGLManager.updateAnnotation(React.findNodeHandle(this.refs[mapRef]), annotation);
+  },
   selectAnnotationAnimated(mapRef, selectedIdentifier) {
```
Comment (bsudekum) [inline index.android.js:25]: Missing new method `removeAnnotation`

**Comment by christopherdro** [comment]
Yea will do that shortly.

File: index.android.js (hunk 1 above)
Comment (bsudekum) [inline index.android.js:25]: Nvm, already there!

Comment (christopherdro) [inline index.android.js:25]: That was actually already in the file. It was missing the native code.

Comment (christopherdro) [inline index.android.js:25]: :+1:

**Comment by bsudekum** [comment]
Tested, feels good. Once example.js, api.md and conflicts are handled, I'll merge this.

**Comment by JonnyBGod** [comment]
+1

**Comment by tsemerad** [comment]
@bsudekum Any update on this? These 2 functions are essential for the app I'm working on. Let me know if there's anything you could use help with on this.

**Pull request closed without merging**